workers/go/throughputstress/workflow.go

```diff
@@ -138,15 +139,17 @@ func ThroughputStressWorkflow(ctx workflow.Context, params *throughputstress.Wor
 				return workflow.ExecuteLocalActivity(localActCtx, activityStub.SelfSignal, ASignal).Get(ctx, nil)
 			},
 			func(ctx workflow.Context) error {
-				// This activity can simulate different traffic patterns as it will sleep for
-				// a configured duration. The duration is randomly selected from a distribution.
-				actInput := MakeSleepInput(params.SleepActivityPerPriority)
-				if actInput == nil {
+				// This activity simulates specific traffic patterns as it will sleep for a configured duration.
+				// The duration is derived from the provided distributions. Optionally with a priority and/or fairness key.
+				actInput := MakeSleepInput(params.SleepActivity, params.SleepActivityPerPriority, params.SleepActivityPerFairness)
+				if actInput.SleepDuration == 0 {
 					return nil
 				}
 				opts := defaultActivityOpts(ctx)
-				opts.Priority = temporal.Priority{
-					PriorityKey: int(actInput.Priority),
+				opts.Priority = actInput.Priority
+				if actInput.FairnessKey != "" {
+					// TODO(fairness): hack until there is a fairness key in the SDK
+					opts.ActivityID = fmt.Sprintf("x-temporal-internal-fairness-key[%s]-%s", actInput.FairnessKey, uuid.New().String())
```

Comment:
This part might be contentious. I don't think anyone outside of the Fairness crew would even consider using this; so I think it's fine/safe.

It would make the Fairness crew's life easier if we could get this PR into main; but if there are strong objections here, we can find another way to run off the branch.